aws/cerberus/cerberus.tf

```diff
@@ -166,6 +172,14 @@ resource "aws_security_group" "cerberus_ec2" {
     description     = "${var.desc_prefix} HTTPS from ALB"
   }
 
+  ingress {
```

Comment:
Should these ingress ports/protocol be driven from variables?

Reply:
I was first thinking that changing the target group port would need to add another ingress. Relooking at it, it just needs to match the target group port. So yes I believe that can be driven from a variable.